biblatex_check.py

```diff
@@ -145,6 +149,17 @@ def open(file, mode='r', buffering=-1, encoding=None,
           "' doesn't exist or is not readable")
     sys.exit(-1)
 
+# Load config file
+if configFile:
+    try:
+        import json5 as json
```

Comment:
What advantages does `json5` have to standard `json`?

Reply:
You can find a detailed description here: http://json5.org
I like best the option to have comments in the file and to not care about trailing commas.